pylearn2/models/mlp.py

```diff
@@ -4455,11 +4473,6 @@ def set_mlp(self, mlp):
         super(FlattenerLayer, self).set_mlp(mlp)
         self.raw_layer.set_mlp(mlp)
 
-    @wraps(Layer.get_weights)
-    def get_weights(self):
-
-        return self.raw_layer.get_weights()
-
```

Comment:
Because it's already defined here: https://github.com/fvisin/pylearn2/blob/55939adf102ecbeb3ac239be86e7960e36b25ad0/pylearn2/models/mlp.py#L756-L762